### PR TITLE
VSFix: Fix README wording issues

### PR DESCRIPTION
**Pull request by ccastrotrejo**
AB#16833229

### CODE REVIEW

**Comment by ecfan** [comment]
@ccastrotrejo: Thank you! I'll review ASAP as I'm trying to close on updating the MQ connector doc because the team is getting pinged frequently by customers.

File: apps/vs-code-designer/src/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Azure Logic Apps for Visual Studio Code (Standard)
 
-Use the Azure Logic Apps (Standard) extension to quickly create, debug, manage, and deploy Logic Apps directly from VS Code.
+In Visual Studio Code, you can use the Azure Logic Apps (Standard) extension to quickly create, debug, manage, and deploy logic apps directly to Azure.
```
Comment (ecfan) [inline apps/vs-code-designer/src/README.md:3]: @ccastrotrejo, here's the full text with the new FW link that I created. The link points to the full steps, so please remove the sections "Create your first logic app" and "Run the logic app locally". Thanks!

```suggestion
In Visual Studio Code, you can use the Azure Logic Apps (Standard) extension to quickly create, debug, manage, and deploy logic apps directly to Azure. For requirements and more information, see [Create a Standard logic app workflow using Visual Studio Code](https://go.microsoft.com/fwlink/?linkid=2222287).
```

**Review by ecfan — CHANGES_REQUESTED** [review]
Answered FW link question with suggested update, thanks!

File: apps/vs-code-designer/src/README.md [hunk 2]
```diff
@@ -1,34 +1,9 @@
...
-1. To start your project, press F5 or the "play" button
-
-2. If you do not have the [Azure Functions Core Tools](https://github.com/Azure/azure-functions-core-tools/releases/tag/4.0.4915) installed, you will be automatically prompted to install. Follow the specified instructions, or skip to the "Deploy" step if you would rather deploy without running locally.
-
-   > TIP: The "Terminal" panel should pop up automatically and you know your project is running if you see output
-
-3. Make sure you have [Azurite Emulator](https://learn.microsoft.com/en-us/azure/storage/common/storage-use-azurite?tabs=visual-studio-code) installed and running. Alternatively, edit the value of `AzureWebJobsStorage` in `local.settings.json` under the project's root folder and under `/workflow-designtime` to a valid connection string for an Azure storage account.
-   > **WARNING**: [Azure Storage Emulator](https://docs.microsoft.com/azure/storage/common/storage-use-emulator) has been deprecated.
-
 ## Deploy to Azure
```
Comment (ecfan) [inline apps/vs-code-designer/src/README.md:7]: Please also remove the "Deploy to Azure" section? Not sure why I didn't see this section when I last reviewed. Thanks!

**Review by ecfan — CHANGES_REQUESTED** [review]
One more...didn't see the "Deploy to Azure" section, which is also already covered in the how-to doc.

File: apps/vs-code-designer/src/README.md (hunk 2 above)
Comment (ccastrotrejo) [reply apps/vs-code-designer/src/README.md:7]: Sure, no problem

**Review by ecfan — APPROVED** [review]
Thank you!

**Review by ecfan — APPROVED** [review]
Thank you!

**Review by Eric-B-Wu — APPROVED** [review]

**Pull request merged**